ThirdPartyAdapters/imobile/imobile/src/main/java/com/google/ads/mediation/imobile/IMobileAdapter.java

```diff
@@ -1,274 +1,274 @@
-package com.google.ads.mediation.imobile;

```

Comment:
Hi Jian, was it resolved correctly? 
see 
https://github.com/googleads/googleads-mobile-android-mediation/compare/master...Vungle:propogate_from_master_af476d3

Reply:
@alexander-gubov it wasn't working on resolving, it there's quite a few conflicts and its a bit hairy to resolve